pages/participate.html

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Participate
+active_nav: participate
+permalink: /participate.html
+---
+
+<div id="section-full">
+    <div class="section-header">
+        <h3>Participate</h3>
+    </div>
+    <div class="section-item">
+        <div class="content">
+            <p>
+                OpenTTD's survey is opt-in; it will not do anything unless you explicitly say you want to participate.
+                If you participate, whenever you leave a game (by loading another, going to the main menu, exiting the game or when the game crashes), the game will fill in the survey, and send it to here.
+            </p>
+            <p>
+                The survey has been designed around a few ground rules:
+                <ul>
+                    <li>No personal data and no data to track a specific user.</li>
+                    <li>The user is in control: you can opt-out at any time, and review the survey result at any time (via "Game Options" in-game).</li>
```

Comment:
```suggestion
                    <li>The user is in control: you can opt-out at any time and review the survey result at any time (via "Game Options" in-game).</li>
```
Alternative line: `The user is in control: you can opt-out or review the survey result at any time (via "Game Options" in-game).`